fenzo-core/src/main/java/com/netflix/fenzo/queues/tiered/Tier.java

```diff
@@ -270,6 +271,46 @@ private String getSortedListString() {
         return b.toString();
     }
 
+    private String getTierCapacityString() {
+        StringBuilder sb = new StringBuilder();
+        if (tierResources != null) {
+            sb.append("Tier ").append(tierNumber).append(" Total Capacity: { ");
```

Comment:
Looks like most of the string builder usage here is for ResAllocs interface objects. Wondering if you'd want to have a default function in that interface to create that string and just use it here for tierResources, effectiveUsedResources, remainingResources, and bucketGuarantees.